Gemfile

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+source 'https://rubygems.org'
+
+git_source(:github) { |repo_name| "https://github.com/#{repo_name}" }
+
+group :development do
+  gem 'fasterer', '~> 0.9.0'
+  gem 'i18n', '~> 1.8.10'
+  gem 'new_super_codebreaker_2021', '~> 0.4.9'
+  gem 'rubocop', '~> 1.7'
```

Comment:
Is the `i18n` a developer dependency?
Same about the `new_super_codebreaker_2021`

Reply:
removed